### ISSUE TITLE
Stop posting math problems

### ISSUE THREAD

**Issue by asarkar** [exercism/rust#1514]
There are numerous problems pulled from math domain that offer **exactly zero** programming knowledge when solved. Examples include problems from project Euler, [Diffie Hellman](https://exercism.org/tracks/rust/exercises/diffie-hellman), and others. We sign up for learning Rust, not for solving math problems. Please stop posing math problem as programming problems even though they may be easier to find on the internet than coming up with original programming questions.

"Skip them if you don't like them" in not a valid answer. I don't want to skip problems, I signed up to learn. Solving math problems doesn't help me learn Rust any more than writing poetry.

Did I say solving math problems don't help me learn Rust?

**Comment by asarkar** [comment]
@iHiD

> added comments that imply the maintainers of different parts of Exercism are stupid, thoughtless or lacking in intelligence compared to yourself

I'm sorry that such a notion was perceived, but none of my comments are a reflection on Exercism members' intelligence, and I've no idea why anyone would think that. Saying a question is not helpful, and providing justification for that argument doesn't mean the author of the question would feel a need to make a complaint. It simply means that the question is not helpful in my opinion, there's no ambiguity there. As you said, Exercism is not obligated to make any changes based on my comment, similarly, I'm entitled to voice my own opinion without disrespecting anyone.

Perhaps I can put it this way: Given the licensing of the website, questions are not any one person's intellectual property, thus, they have no reason to be intellectually offended if someone question the content of the questions.

**Comment by iHiD** [comment]
> I'm sorry that such a notion was perceived

Thank you. 

>  I've no idea why anyone would think that

People would think that because you're stating your opinion in such strong absolute terms, rather than stating things are opinions or asking questions. For example take these three examples:
> that offer exactly zero programming knowledge when solved
> but it is a waste of time for the students looking to learn a programming language.
> Without directory support, all this PR does is help close a ticket, not help a student.

Why has someone done any of the work to create these exercises or PR? The way you are communicating suggests that it is ridiculous and pointless that they have done that, which therefore suggests that they are clearly not as good at thinking as you. That's how multiple people have interpreted your comments across multiple repos now.

> I'm entitled to voice my own opinion without disrespecting anyone

Yes, but if multiple people feel disrespected by you, then you're likely not abiding by the Code of Conduct, which states we must all be empathetic, welcoming, kind, honest, supportive and polite. 

The third example (from the CLI PR) you dismissed someone's work in a rude and dismissive manner, making a statement that the PR helped no-one, which added no value to the conversation. You didn't support the PR creator, you weren't kind or empathetic, or polite to them. I understand you may not have intended to be rude, but you definitely came across as rude to lots of people who read that PR, and to me.

> Given the licensing of the website

The licensing of the website describes how you may **use** the code in it, it does not define how you may communicate around it. Our Code of Conduct does that, and we enforce our Code of Conduct strictly. People work hard on Exercism for free and it's my job to ensure that they are spoken to respectfully.

---

In summary, when engaging in discussion inside Exercism its important you consider how you might be making other people feel, and remember that people from different cultures have different perceptions on what is polite and kind, so please edge on the side of being overly kind rather than not kind enough. Thank you.
